script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -1,82 +1,82 @@
 <Page1>
 Only a few minutes have passed since I stopped by the convenience store.
-It is now half an hour before my usual time to go to school.
+It's now half an hour before I usually go to school.
 <Page2>
-The students walking on the sidewalk with their school uniforms are sparse.
-In about ten minutes, the main gate will be crowded with students, but for now there are only few of them including myself.
-There is also no sign of the beautiful woman sitting on the guardrail, waiting for someone like in that time.
+There aren't many students visible on the sidewalk.
+In about ten minutes, the main gate will be crowded with students, but for now there are only two or three of them, myself included.
+There's also no sign of the unbelievably beautiful woman sitting on the guardrail, awaiting someone like that time.
```

Comment:
> あの時のように、ガードレールに腰を下ろして誰かを待っている、嘘みたいに美しかった女の姿もない。
Unlike that time, there's no sign of the unbelievably beautiful woman sitting on the guardrail, waiting for someone.